NewForce-Cohort-5/tabloidfullstack-actual-spring
# View list of categories

Trello #9 View All Categories

This code will allow you to click the category navbar and be directed to the category page which then lists the categories from the database in alphabetic order.

A mode, controller, interface, and repository for Category has been added to the back end code.
Category Card, category list, category provider has been added to allow for the viewing of the categories in the browser.

Type of change
New feature (non-breaking change which adds functionality).
This change requires a documentation update.

Testing Instructions
Reference How to test a teammate's PR
Login using foo@bar
Go to the category management tab on the navbar
You should be able to view the categories listed in alphabetic order.

Make sure to add all seed data before testing.
Run this command below to verify the categories match the database.
Select * From Category

Please cross check database to browser to ensure the right information is being displayed properly.


- [ X] New feature (non-breaking change which adds functionality)
- [X ] This change requires a documentation update
